add azure key vault secret springboot starter dependencies in the document (#512)
MicrosoftDocs/azure-dev-docs: articles/java/spring-framework/configure-spring-boot-starter-java-app-with-azure-key-vault.md

1. Kill the process that's running from `mvn spring-boot:run`. You can type Ctrl-C, or you can use the `jps` command to get the pid of the `Launcher` process and kill it.

## Create the app without Spring Initializr

> This section shows how to include azure key vault secrets to your existing spring boot project without using Spring Initializr.

Reviewer: ```suggestion
This section shows how to include Azure Key Vault secrets to your existing Spring Boot project without using Spring Initializr.
```

> Just as what Spring Initializr would have done, please add the following to your current *pom.xml* file.

Reviewer: ```suggestion
To manually add the same the configuration that Spring Initializr generates, add the following configuration to your *pom.xml* file.
```
